### PR TITLE
Sam/cart components

### PR DESCRIPTION
**Pull request by orizabal**
Made Cart and LineItem components

### CODE REVIEW

File: src/components/Cart/LineItem.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                            </button>
+                        </div>
+                    </div>
+                )})
+            }
+        </li>
+    );
+}
+
+export default LineItem;
```
Comment (michaelmeng998) [inline src/components/Cart/LineItem.js:61]: [nitpick] adding a newline in all files

File: src/components/Cart/Cart.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        if ( cartStatus === true ) {
+            button.classList.add( "hide" );
+        } else {
+            button.classList.remove( "hide" );
+        }
+
+        function getCount() {
+            let lineItems =
+                checkoutState.lineItems && checkoutState.lineItems.length > 0
+                ? checkoutState.lineItems
```
Comment (michaelmeng998) [inline src/components/Cart/Cart.js:43]: Is there ever a case where checkoutState.lineItems.length > 0 but checkoutState.lineItems object is NULL?

**Review by michaelmeng998 — COMMENTED** [review]
looking good, just had some question!

File: src/components/Cart/Cart.js (hunk 2 above)
Comment (orizabal) [reply src/components/Cart/Cart.js:43]: > Is there ever a case where checkoutState.lineItems.length > 0 but checkoutState.lineItems object is NULL?

I don't think so but the example that Shopify had checked for both cases

Comment (michaelmeng998) [reply src/components/Cart/Cart.js:43]: I see, all good in that case.

**Review by faizaanmadhani — APPROVED** [review]
This looks pretty good. How does it relate to your other PR though?

**Pull request closed without merging**